### PR TITLE
Track navigation

### PR DESCRIPTION
**Pull request by descartesholland**

### CODE REVIEW

File: dancer.html [hunk 1]
```diff
@@ -43,20 +76,24 @@
...
-    
-    <div id ="video">
-
     </div>
-    <div id ="seekbar">
+    <div id="content" style="width: 640; margin: 0 auto">
+      <div id ="video">
 
+      </div>
+      <div id ="seekbar">
```
Comment (anpere) [inline dancer.html]: The seekbar is gets covered by the autocomplete in search.

Comment (anpere) [reply dancer.html]: though this might not be a problem when we merge into master.

Comment (descartesholland) [reply dancer.html]: Should not be a problem in master (check commit before last)

**Comment by descartesholland** [comment]
Should be ready for merge. Video content is now centered and adjusts sort of dynamically @anpere

File: dancer.html [hunk 2]
```diff
@@ -11,36 +11,70 @@
...
+      }
+      $("#indicator").css({ position: 'relative',
+                            left: clickedPosX_centered });
+      
+      var indicator = $("#indicator");
+
+      var ui = {};
+      ui['helper'] = indicator;
+      ui['position'] = indicator.position();
+      ui['offset'] =  {top: 0, left: 0};
```
Comment (anpere) [inline dancer.html]: where is ui used?

File: mainLayout.css [hunk 3]
```diff
@@ -1,41 +1,92 @@
...
 
-
+#indicator {
+    background: #FF0000;
+    height: 33px;
+    width: 6px;
+    align-self: flex-end;
+}
 
 #video{
```
Comment (anpere) [inline mainLayout.css:81]: Could we make video the same size as the trackbar?

**Review by anpere — COMMENTED** [review]
Looks good to me. Just had a few questions that I posted as comments

File: dancer.html (hunk 2 above)
Comment (descartesholland) [reply dancer.html]: I'm not sure, actually.. I think I removed its use. Good find

File: mainLayout.css (hunk 3 above)
Comment (anpere) [reply mainLayout.css:81]: Oops, not actually an issue in the current iteration.

**Pull request merged**